bin/improver-nbhood-land-and-sea

```diff
@@ -50,11 +50,11 @@ def main():
     """Load in arguments for applying neighbourhood processing when using a
     mask."""
     parser = ArgParser(
-        description='Neighbourhood the input cube as two distinct regions of '
-        'land and sea. If performed as a single level neighbourhood, a land-'
-        'sea mask should be provided. If instead topographic_zone '
+        description='Neighbourhood the input dataset over two distinct regions'
+        ' of land and sea. If performed as a single level neighbourhood, a '
+        'land-sea mask should be provided. If instead topographic_zone '
         'neighbourhooding is being employed, the mask should be one of '
-        'topographic zones. In the latter case a weights cube is also required'
+        'topographic zones. In the latter case a weights array is also needed'
         ' to collapse the topographic_zone coordinate. These weights are '
```

Comment:
There are some references to cubes in the exceptions raised below this, although I think the issue was really referring to the CLI help, so these are probably fine..

Reply:
(Related - wind-downscaling also references a cube in an exception raised in the CLI script. Again, I'm not sure if this is a problem, because exceptions ultimately raised in the non-CLI python code will almost certainly refer to cubes.)

Reply:
I'd rather not change the exception messages as this is likely to require updates to unit tests (eg where `assertRaisesRegexp` is used).  We could (and probably should) open follow on work to review exception messages though (as a whole, not just references to cubes).

Reply:
(I'm not so sure. Are the CLIs covered by unit tests? I always considered the BATS tests as a sort of hybrid unit/acceptance test - unit in the case where we're not passing any data in, and are just looking at the printed output.)